test/fluree/db/json_ld/api_test.cljc

```diff
@@ -274,4 +274,102 @@
            (is (= (get-in db [:schema :context])
                   (get-in loaded-db [:schema :context])))
            (is (= (get-in db [:schema :context-str])
-                  (get-in loaded-db [:schema :context-str]))))))))
+                  (get-in loaded-db [:schema :context-str]))))))
+
+     (testing "can load a ledger with `list` values"
+       (with-tmp-dir storage-path
```

Comment:
Yeah, I chose file conns here because I wanted to replicate the setup http-api-gateway was using when I found this bug in the first place. And because this batch of `load` tests was already here and who can resist a good copypasta? 😄